pages/locations/index.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import { useCampuses, useCurrentBreakpoint } from 'hooks';
+import { kebabCase } from 'lodash';
+import { useState } from 'react';
+import {
+  Box,
+  CardGrid,
+  Loader,
+  Image,
+  Button,
+  TextInput,
+  HorizontalHighlightCard,
+} from 'ui-kit';
+import { Layout, CustomLink, Video } from 'components';
+
+const FindNearestLocation = () => {
+  const [results, setResults] = useState([{ geometry: { location: {} } }]);
+  const [address, setAddress] = useState();
+  const currentBreakpoint = useCurrentBreakpoint();
+
+  const userCoordinatesExists =
+    results[0]?.geometry?.location?.lat && results[0]?.geometry?.location?.lng;
+
+  const { campuses, loading, refetch } = useCampuses(
+    userCoordinatesExists && {
+      variables: {
+        latitude: results[0].geometry.location.lat,
+        longitude: results[0].geometry.location.lng,
+      },
+    }
+  );
+
+  // sort campuses by distance from location
+  let sortedCampuses = [...campuses].sort((a, b) => {
+    return a.distanceFromLocation - b.distanceFromLocation;
+  });
+
+  // Remove Online Campus from sortedCampuses array to ensure it is always the first item in the list
+  const onlineCampusIndex = sortedCampuses.findIndex(campus => {
+    return campus.name === 'Online (CF Everywhere)';
+  });
+  const onlineCampus = sortedCampuses[onlineCampusIndex];
+  sortedCampuses.splice(onlineCampusIndex, 1);
+
+  const getCoordinates = async () => {
+    if (address) {
+      // get coordinates from address
+      const response = await fetch(`/api/googlemaps?place=${address}`);
+      const data = await response.json();
+      setResults(data.results);
+    } else {
+      // if no address is entered, set results to an empty array
+      setResults([{ geometry: { location: {} } }]);
+    }
+  };
+
+  return (
+    <Layout>
+      <Image
+        source={'external-landing/new-here-1.jpeg'}
```

Comment:
@leoalfonso14 We want to make this a background video(_we can use the homepage video while we wait_), we still also need to add the text overlay as seen below:
![image](https://github.com/christfellowshipchurch/web-app-v2/assets/46049974/02f4dafa-49cc-4837-aaa4-a19144e24e03)